client/components/payment-methods-checkboxes/test/index.js

```diff
@@ -220,7 +228,9 @@ describe( 'PaymentMethodsCheckboxes', () => {
 		);
 
 		expect( page.container ).not.toContainHTML(
-			'<span class="wcpay-pill payment-status-inactive">Contact WooCommerce Support</span>'
+			'<span class="wcpay-pill payment-status-inactive">' +
+				'<span class="woocommerce-pill css-1qmnemh-Text e15wbhsk0">' +
+				'Contact WooCommerce Support</span></span>'
```

Comment:
After chatting IRL, this test needs to be updated because it's checking that HTML doesn't exist but the HTML given in this case is wrong so this tests essentially says: make sure this incorrect HTML isn't included when it's trying to test that the "Contact support" pill isn't shown in this circumstance. 

To make this test a little more robust you could change this test to something like: 


```js
// Test that the Contact support pill content isn't shown
expect( queryByText( 'Contact WooCommerce Support' ) ).not.toBeInTheDocument();
```

Reply:
Fixed in [32fbc6e](https://github.com/Automattic/woocommerce-payments/pull/5890/commits/32fbc6ee8d4cefa525e0d0d9614e44f8586b1128)